internal/command/jsonstate/state.go

```diff
@@ -48,9 +48,10 @@ type StateValues struct {
 }
 
 type Output struct {
-	Sensitive bool            `json:"sensitive"`
-	Value     json.RawMessage `json:"value,omitempty"`
-	Type      json.RawMessage `json:"type,omitempty"`
+	Sensitive  bool            `json:"sensitive"`
+	Deprecated string          `json:"deprecated,omitempty"`
```

Comment:
This is the field we may want to document somewhere

Reply:
I will extend [the json output format page](https://opentofu.org/docs/internals/json-format/), thanks!

Reply:
This is covered in https://github.com/opentofu/opentofu/pull/2706